1.9.py

```diff
@@ -0,0 +1,14 @@
+#Exercise 2.1
+answer1 = 42*60+42
+print(f'There are {answer1} seconds in 42 minutes 42 seconds\n')
+
+#Exercise 2.2
+answer2  = 10/1.61
+print (f'There are {answer2} miles in 10 kilometers\n')
+
+#Exercise 2.3
+distance_km=10
+time=42.42/60
+distance_mile = distance_km/1.61
+answer3 = distance_mile/time
+print(f'The average speed is {answer3} kilometers per hour\n')
```

Comment:
in 1.9.3 There are two questions, you printed only one ;)
And please read the second question - it is "miles per hour, not kilometers"

Reply:
Done! Thank you